config/jobs/periodic/containerd/test-containerd-powervs-periodic.yaml

```diff
@@ -0,0 +1,66 @@
+periodics:
+  - name: periodic-containerd-test-powerVS
+    cluster: k8s-ppc64le-cluster
+    decorate: true
+    decoration_config:
+        timeout: 4h
+    interval: 24h
+    spec:
+      nodeSelector:
+        dedicated: runtimesTeam
+      tolerations:
+      - key: dedicated
+        operator: "Equal"
+        value: runtimesTeam
+        effect: "NoSchedule"
+      volumes:
+      - name: atos-ssh-volume
+        secret:
+          secretName: atos-powervs-ssh-key
+          defaultMode: 256
+      containers:
+        - image: quay.io/powercloud/all-in-one:0.3
+          volumeMounts:
+          - name: atos-ssh-volume
+            readOnly: true
+            mountPath: "/etc/ssh-volume"
+          command:
+            - /bin/bash
+            - -c
+            - |
+              set -o errexit
+              set -o nounset
+              set -o pipefail
+              set -o xtrace
+
+              # Connect to IBMCloud
+              echo "" | ibmcloud login
+              ibmcloud target -r eu-gb -g runtimes-dev-resource-group
+              ibmcloud pi service-target crn:v1:bluemix:public:power-iaas:lon06:a/65b64c1f1c29460e8c2e4bbfbd893c2c:58482da9-46b2-411b-b9fb-ad4efe6be307::
+
+              wget https://raw.githubusercontent.com/ppc64le-cloud/docker-ce-build/main/test-containerd/instanciate_powervs_vm.sh
+              chmod +x instanciate_powervs_vm.sh
+
+              # Run test server
+              mkdir ${ARTIFACTS}/runc_l1/
+              bash -x ./instanciate_powervs_vm.sh --output ${ARTIFACTS}/runc_l1/ --name rdr-runtimes-dev-etienne_guesnet-runc-l1 --runc runc --runtime "io.containerd.runtime.v1.linux" --key atos-key --network  'public-192_168_139_176-29-VLAN_2050' 2>&1 | tee runc_l1.txt
```

Comment:
The VM names need to be changed.
Instead of a person's name(Seems [etienne.guesnet](https://ibm.enterprise.slack.com/team/WMPTVUFHC) has left the organization) in the string, I suggest the job related word to be present in the VM name.
May be something like `rdr-runtimes-dev-**periodic-containerd**-runc-v1`

Reply:
I have updated the VM names as suggested. I will move on to debugging the test failures.